arch/configure.defaults

```diff
@@ -2068,6 +2068,49 @@ NETCDFPAR_BUILD	=      CONFIGURE_NETCDFPAR_BUILD
 
 #insert new stanza here
 
+###########################################################
+#ARCH    Linux aarch64, NVHPC compiler with nvc # serial smpar dmpar dm+sm
+#
+DESCRIPTION     =       NVHPC ($SFC/$SCC)
+DMPARALLEL      =       # 1
+OMPCPP          =       # -D_OPENMP
+OMP             =       # -mp -Minfo=mp -Mrecursive
+OMPCC           =       # -mp
+SFC             =       nvfortran
+SCC             =       nvc
+CCOMP           =       nvc
+DM_FC           =       mpif90
+DM_CC           =       mpicc
+FC              =       CONFIGURE_FC
+CC              =       CONFIGURE_CC
+LD              =       $(FC)
+RWORDSIZE       =       CONFIGURE_RWORDSIZE
+PROMOTION       =       -r$(RWORDSIZE) -i4
+ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM
+CFLAGS_LOCAL    =       -w -O3
+LDFLAGS_LOCAL   =
+CPLUSPLUSLIB    =
+ESMF_LDFLAG     =       $(CPLUSPLUSLIB)
+FCOPTIM         =       -O3 #-fastsse -Mvect=noaltcode -Msmartalloc -Mprefetch=distance:8 -Mfprelaxed # -Minfo=all =Mneginfo=all
```

Comment:
Would all of these options be applicable to `aarch64` if a user were to uncomment them?

Reply:
Mostly yes, other than fastsse which is x86 specific. Msmartalloc seems to be deprecated.  The other flags are OK,
The -fastsse flag equivalent is -fast.

Reply:
In this case, I'd suggest cleaning up these flags by removing deprecated flags or flags that don't apply to aarch64. Essentially, I think everything in this new stanza should be tailored to aarch64.

Reply:
I cleaned up and tested the flags on ARM. Pushed a new version. I'm keeping fprelaxed there but commented out as it does  change the output slightly (at least over a 24 hour run).